LL/ciarraige_luachra.ttl

```diff
@@ -19,107 +19,125 @@
     foaf:name "Conchobuir";
     foaf:name "Conchobur";
-    rel:childOf <#Cathail>.
+    rel:childOf <#Cathail>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Conchobuir>.
 
 <#Cathail>
     a foaf:Person;
     foaf:name "Cathail";
     foaf:name "Cathal";
-    rel:childOf <#Conchobuir-4a65ed99>.
+    rel:childOf <#Conchobuir-4a65ed99>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Cathail>.
 
 <#Conchobuir-4a65ed99>
     a foaf:Person;
     foaf:name "Conchobuir";
     foaf:name "Conchobur";
-    rel:childOf <#Muridaig>.
+    rel:childOf <#Muridaig>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Conchobuir-3c5b442e>.
 
 <#Muridaig>
     a foaf:Person;
     foaf:name "Muridaig";
     foaf:name "Muridach";
-    rel:childOf <#Diarmata>.
+    rel:childOf <#Diarmata>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Muridaig>.
 
 <#Diarmata>
     a foaf:Person;
     foaf:name "Diarmata";
     foaf:name "Diarmait";
-    rel:childOf <#Cormaic>.
+    rel:childOf <#Cormaic>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Diarmata>.
 
 <#Cormaic>
     a foaf:Person;
     foaf:name "Cormaic";
     foaf:name "Cormac";
-    rel:childOf <#Indrechtaig>.
+    rel:childOf <#Indrechtaig>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Cormaic>.
 
 <#Indrechtaig>
     a foaf:Person;
     foaf:name "Indrechtaig";
     foaf:name "Indrechtach";
-    rel:childOf <#Aeda>.
+    rel:childOf <#Aeda>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Inrechtaig>.
 
 <#Aeda>
     a foaf:Person;
     foaf:name "Aeda";
     foaf:name "Aed";
-    rel:childOf <#Cobthaig>.
+    rel:childOf <#Cobthaig>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Aeda>.
 
 <#Cobthaig>
     a foaf:Person;
     foaf:name "Cobthaig";
     foaf:name "Cobthach";
-    rel:childOf <#MaelChoba>.
+    rel:childOf <#MaelChoba>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Cobthaig>.
 
 <#MaelChoba>
     a foaf:Person;
     foaf:name "Mael Choba";
     foaf:name "Mael Cob";
-    rel:childOf <#FhlaindFeorna>.
+    rel:childOf <#FhlaindFeorna>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#MaelChoba>.
 
 <#FhlaindFeorna>
     a foaf:Person;
     foaf:name "Fhlaind Feorna";
     foaf:name "Flan Feorn";
-    rel:childOf <#Colmain>.
+    rel:childOf <#Colmain>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#FlaindEorna>.
 
 <#Colmain>
     a foaf:Person;
     foaf:name "Colmain";
     foaf:name "Colman";
-    rel:childOf <#Rechtabrat>.
+    rel:childOf <#Rechtabrat>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Colmain>.
 
 <#Rechtabrat>
     a foaf:Person;
     foaf:name "Rechtabrat";
-    rel:childOf <#MaeleTuile>.
+    rel:childOf <#MaeleTuile>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#Rectabrat>.
 
 <#MaeleTuile>
     a foaf:Person;
     foaf:name "Maele Tuile";
     foaf:name "Mael Tul";
-    rel:childOf <#Aidloga>.
+    rel:childOf <#Aidloga>;
+    owl:sameAs <http://example.com/LL/ciarraige.ttl#MaeliTuile>.
```

Comment:
These are pointing to a completely different name.  Are you sure this is correct?